services/google_search_service/job_enqueuer_test.go

```diff
@@ -0,0 +1,72 @@
+package google_search_service
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/gocraft/work"
+	"github.com/gomodule/redigo/redis"
+	"github.com/gutakk/go-google-scraper/db"
+	"github.com/gutakk/go-google-scraper/models"
+	testDB "github.com/gutakk/go-google-scraper/tests/db"
+
+	"github.com/stretchr/testify/suite"
+	"gopkg.in/go-playground/assert.v1"
```

Comment:
I'm not sure about the order here, as I understand the `"github.com/gocraft/work"`, `"github.com/gomodule/redigo/redis"` also the external lib, should we move it below @gutakk

Reply:
Fixed in 162033b0a7ad6424a236310201baad3a5cc97003